src/main.rs

```diff
@@ -1139,6 +1186,48 @@ fn find_device() -> Result<PathBuf> {
     Ok(path)
 }
 
+fn event_handler(data: &[u8]) -> ::std::os::raw::c_int {
+    static START_TIME_ONCE: Once = Once::new();
+    static mut START_TIME: Option<Instant> = None;
```

Comment:
I'm wondering - have we simply reached the point where we need a `struct HidRecorder` which contains the `start_time` and we pass that into the event handler and `parse_input_report()` (so we share the same start time instead of initializing it twice). That would make life a lot easier...

That struct can also contain whatever other values we are forcibly trying to make `static` with a lot of effort.

Reply:
ended up doing this with a `OnceCell` instead, see #34